clms-api/src/main/resources/db/migration/V10__Create_course_table.sql

```diff
@@ -1,14 +1,13 @@
 create table if not exists courses(
     id serial primary key,
     name varChar(225) not null,
-    description varChar(225) not null,
-)
+    description varChar(225) not null
+);
 
-create table if not exists course_members(
+create table if not exists course_members (
     user_id int not null,
     course_id int not null,
     primary key (user_id, course_id),
     foreign key (user_id) references users(id),
     foreign key (course_id) references courses(id)
-
-)
\ No newline at end of file
+);
```

Comment:
- sql syntax errors
- We already had question bank V7 file so i moved this to V10
- Migration file name schema: V<Version number><double underscore><title in snake case>.sq